web/templates/left_sidebar.hbs

```diff
@@ -186,11 +186,16 @@
         </div>
 
         <div id="streams_list" class="zoom-out">
-            <div class="stream_search_section left-sidebar-filter-row">
-                <input class="stream-list-filter home-page-input filter_text_input" type="text" autocomplete="off" placeholder="{{t 'Filter channels' }}" />
-                <button type="button" class="clear_search_button" id="clear_search_stream_button">
-                    <i class="zulip-icon zulip-icon-close" aria-hidden="true"></i>
-                </button>
+            <div id="stream_search_and_add" class="zoom-in-hide">
```

Comment:
Remember to use dashes, not underscores, whenever you're picking a new variable name.

Reply:
I've updated a few and will try to remember this next time! Thanks